src/styles/main.scss

```diff
@@ -0,0 +1,75 @@
+$item-bg-color: #eee;
+$calendar-item-size: 100px;
+$calendar-gap: 1px;
+$calendar-item-width: $calendar-item-size + $calendar-gap;
+$week: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
```

Comment:
The universal selector `*` is used to reset default margins and paddings, which is fine, but it's a heavy selector and might cause performance issues on large documents. Consider using a more specific selector or applying these styles to body elements only if that's sufficient.